### ISSUE TITLE
Add support for using multiple dictionaries / languages

### ISSUE THREAD

**Issue by Bisaloo** [ropensci/spelling#73]
There is a growing interest in multilingual R packages, and this relates to recent [rOpenSci work](https://ropensci.org/multilingual-publishing/).

Would it be possible to add the ability to spellcheck a package with multiple languages? To address, e.g., the case when documentation or a README is provided in two languages ([real-life example](https://github.com/Bisaloo/semaforos)). I understand that this might then miss some typos because they are a valid word in the other languages.

From a quick glance, it looks like this might require a change to the hunspell (this change? https://github.com/ropensci/hunspell/pull/37) so feel free to move there if you believe that's more appropriate.

**Comment by Bisaloo** [comment]
There is also this related question which suggests interest in this feature: https://stackoverflow.com/questions/50168526/use-multiple-dictionaries-using-hunspell-in-r

**Comment by jeroen** [comment]
I'm not a linguist but I think the concept of "multiple languages" is a bit ambiguous. Would you expect it to be able to spellcheck a document with a mix of languages and accept e.g. a chineese word pluralized with to english grammar?

**Comment by Bisaloo** [comment]
Yes, I think you make a good point about using multiple languages in a single document. But let's say I want to use different languages in two different vignettes. How would you go with using spelling in this situation?
